servers/jak/webservice/app/src/main/java/com/couchbase/lite/javaws/mobiletest/EmbeddedJettyServer.java

```diff
@@ -0,0 +1,22 @@
+package com.couchbase.lite.javaws.mobiletest;
```

Comment:
I do not think this should be a new class/file.  I think that the `main` method should just be added to `TestServerApp.java` similar to the way the main method is set up in the desktop version

Reply:
Done!